contents/handbook/growth/marketing/seo-guide.md

```diff
@@ -0,0 +1,110 @@
+---
+title: SEO best practices
+sidebar: Handbook
+showTitle: true
+---
+
+## General principles
+
+### 1. Think about search intent when choosing what to write
+
+Successful SEO is all about anticipating a searcher's desire and delivering the right answers for them. This means thinking beyond just the initial search term – e.g. someone searching for the difference between retention rate and churn has a deeper desire: "I want to improve retention/churn."
+
+### 2. Get to the point
+
+Google increasingly rewards pages that answer questions quickly. It's annoying for readers to have to scroll through masses of filler to find the answer to their question. Bear this in mind when writing.
+
+### 3. Make guides easy to scan
+
+Readers rarely just start reading an article from the start – especially if they're coming from search. They're likely to scan the page looking for signs of quality and reassurance the page will answer their question. Clear headings and engaging visuals (e.g. diagrams and tables) massively help here.
+
+### 4. Write headlines that stand out
+
+Writing headlines that work for users and Google is hard. The best thing we can do is try to find ways to make our headlines stand out in some way. Does every result for a search show articles with nearly the same headline? Do something completely different. Make it shorter, more opinionated. Anything that's different is good.
+
+### 5. There's always more than one search term
+
+Good SEO articles always target more than one search term.
+
+While you may start with a core search term in mind, remember there are always multiple ways to search for the same information. Sometimes it's better to target a similar but lower volume search term than the big obvious one.
```

Comment:
```suggestion
While you may start with a core search term in mind, remember there are always multiple ways to search for the same information. Sometimes it's better to target a similar but lower volume search term than the big obvious one.

For example, the parent search term "user persona" (27,000/mo) has numerous more specific derivations:

- Define user persona (8,100)
- Create user persona (3,600) 
- Persona modelling (720)
- Benefits of personas (50) 
- User persona examples (5,400) 
- Examples of user persona (260)
- How to create personas (2,900)
- What is a user persona (260)
- User persona template (27,000) 

It may be very hard to rank for the most popular term, but many of these smaller and small specific terms are more realistic.
```